server/ladder_service.py

```diff
@@ -467,10 +467,16 @@ def game_options(player: Player) -> GameLaunchOptions:
                 ])
             await game.wait_launched(60 + 10 * len(all_guests))
             self._logger.debug("Ladder game launched successfully")
-        except Exception:
+        except Exception as e:
+            if isinstance(e, asyncio.TimeoutError):
+                self._logger.info(
+                    "Failed to start ladder game! Game setup timed out"
+                )
+            else:
+                self._logger.exception("Failed to start ladder game!")
```

Comment:
Similar comment as above, I think in including the player ids/names could be useful here.

Reply:
Yea that's probably true